README

```diff
@@ -57,10 +57,22 @@ EGLWLMockNavigation:
    Syntax:  [wayland_display_to_connect_to] <your installation path>/bin/EGLWLMockNavigation
    Example: WAYLAND_DISPLAY=wayland-1 $HOME/bin/EGLWLMockNavigation
 
-How to test
+How to test functional testing
 ====================================
 1. Build the testsuite by setting BUILD_ILM_API_TESTS option.
    Example: cmake -DBUILD_ILM_API_TESTS
 2. After starting up Weston run the testsuite.
    Syntax:  [wayland_display_to_connect_to] <your installation path>/bin/ivi-layermanagement-api-test
    Example: WAYLAND_DISPLAY=wayland-1 $HOME/bin/ivi-layermanagement-api-test
+
+How to test unit testing
+====================================
+1. Build the testsuite by setting BUILD_ILM_UNIT_TESTS option.
+   Example: cmake -DBUILD_ILM_UNIT_TESTS=ON
+2. After build successfully, just run the testsuite right away.
+   Syntax: <your installation path>/bin/unittest*
+   Example: $HOME/install/bin/unittest-ivi-controller
+3. (Optional) To display code coverage results, run shell script in unittest folder
+   Syntax: code_coverage.sh <your build path>
```

Comment:
This require the build path to script, it means the COV flags are default in the Cmake of tests, I shouldn't expected this. the binary when running will dump the *.gcno files in somewhere, it may get permission errors if the user don't have allow to write the files

Reply:
Yes, this suggestion is good. I will apply it to the code.